information_msg information
position_msg position
string emotion